doc/md/fundamentals/1-basic-syntax/5-characters-strings.md

```diff
@@ -6,9 +6,9 @@ sidebar_position: 5
 
```

Comment:
The file heading should probably be `# Characters and Text`

(just because Motoko calls the string type  `Text`)